master/getting-started/kubernetes/upgrade/test.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Testing the data migration
+---
+
+1. Run `calico-upgrade dry-run` to validate the migration.
```

Comment:
Data migration is a very different process than upgrade.

Reply:
Oh this testing procedure is broken out because I'll need to discuss the errors that they may encounter and how to read the reports that get generated, etc. But right now, it's not much here....I expect it will be filled out much more in the coming days.